src/pages/home/report/ReportActionsList.tsx

```diff
@@ -495,6 +495,28 @@ function ReportActionsList({
         calculateUnreadMarker();
     }, [calculateUnreadMarker, report.lastReadTime, messageManuallyMarkedUnread]);
 
+    useEffect(() => {
+        const scrollToFirstUnreadMessage = () => {
+            if (!currentUnreadMarker) {
+                return;
+            }
+
+            const unreadMessageIndex = sortedVisibleReportActions.findIndex((action) => action.reportActionID === currentUnreadMarker);
+
+            if (unreadMessageIndex !== -1) {
+                // We're passing viewPosition: 1 to scroll to the top of the
+                // unread message (marker) since we're using an inverted FlatList.
+                reportScrollManager?.scrollToIndex(unreadMessageIndex, false, 1);
```

Comment:
I think this has a side effect where the new message received automatically scrolls to the index of the unread message. 


https://github.com/Expensify/App/assets/25520267/31b3b3aa-42d5-4f27-9403-a5e3691b10f4

Reply:
@mollfpr Yeah, I was thinking about adding an additional check such that if the report is open and the message is within the visible view -> don't scroll. But I was not sure exactly what's the expected behaviour so I didn't add that check.

This behaviour looks good to me though.
Should we only scroll the first time when we open the report and there are new messages and NOT when the report is already opened ?

Reply:
>Should we only scroll the first time when we open the report and there are new messages and NOT when the report is already opened ?

I guess this is the correct behavior. Currently, in production, the list is not scrolled when a new message is received. The case that might be needed when the user is reading a long message and new message comes in, the list shouldn't scrolled here.

Reply:
Understood, will update the code and let you know when it's ready!

Reply:
@mollfpr Done, just pushed the changes to include the scroll offset check. The mentioned issue should be fixed, let me know if it tests well so we can move forward with the review!